modules/jlink/artifacts/opt/jboss/container/java/s2i/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM    registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
I don't think we need this file

Reply:
New path in current PR is modules/jlink/artifacts/opt/jboss/container/java/jlink/Dockerfile - file not needed